### PR TITLE
Updated timers to match betaflight in preparation for F4

### PR DESCRIPTION
**Pull request by martinbudden**
Also added some snippets of F4 code under `#defines`.

Tidied whitespace and `#includes` in `target.c` files.

### CODE REVIEW

File: src/main/drivers/pwm_output.c [hunk 1]
```diff
@@ -68,7 +68,7 @@ static void pwmOCConfig(TIM_TypeDef *tim, uint8_t channel, uint16_t value)
     TIM_OCInitStructure.TIM_OutputState = TIM_OutputState_Enable;
     TIM_OCInitStructure.TIM_OutputNState = TIM_OutputNState_Disable;
     TIM_OCInitStructure.TIM_Pulse = value;
-    TIM_OCInitStructure.TIM_OCPolarity = TIM_OCPolarity_Low;
+    TIM_OCInitStructure.TIM_OCPolarity = ouputPolarity ? TIM_OCPolarity_High : TIM_OCPolarity_Low;
```
Comment (digitalentity) [inline src/main/drivers/pwm_output.c]: `ouputPolarity` - maybe it's a typo here?

File: src/main/target/ALIENFLIGHTF3/target.c [hunk 2]
```diff
@@ -64,37 +63,28 @@ const uint16_t airPWM[] = {
 
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    //
     // 6 x 3 pin headers
-    //
+    { TIM15, IO_TAG(PB15), TIM_Channel_2, TIM1_BRK_TIM15_IRQn,     1, IOCFG_AF_PP, GPIO_AF_1 }, // PWM1  - PB15 - TIM1_CH3N, TIM15_CH1N, *TIM15_CH2
```
Comment (digitalentity) [inline src/main/target/ALIENFLIGHTF3/target.c:67]: I think `1` and `0` here should be replaced by proper names? What do you think?

Comment (martinbudden) [inline src/main/target/ALIENFLIGHTF3/target.c:67]: I presume you mean in all targets.

Yes,  eventually we should change that, but it's a big edit, so I don't think it should hold up this merge.

Once this merge is in the `target.c` files will be **identical** to the files in betaflight - so we can have confidence that they have been tested.

Comment (digitalentity) [inline src/main/target/ALIENFLIGHTF3/target.c:67]: Having same target.c files between BF and INAV would be nice!

Comment (martinbudden) [inline src/main/target/ALIENFLIGHTF3/target.c:67]: The aim is to have the same target.c files, the same drivers and, very similar target.h files.

**Pull request merged**